tailwind/plugins/theme/gc-fontFamily.js

```diff
@@ -5,7 +5,8 @@ module.exports = function () {
   return {
     'sans': ['var(--font-source-sans)', 'sans-serif'],
     'serif': ['var(--font-source-serif)', 'serif'],
-    'druk': ['"Druk Web"', 'sans-serif'],
-    'druk-wide': ['"Druk Wide Web"', 'sans-serif'],
+    'druk': ['var(--font-druk)', 'sans-serif'],
+    'druk-wide': ['var(--font-druk-wide)', 'sans-serif'],
+    'stanford': ['var(--font-stanford)', 'serif'],
```

Comment:
I've decided to host the Stanford font locally, since Nextjs has good font optimization and automatically preloaded
https://nextjs.org/docs/pages/building-your-application/optimizing/fonts